docs/release-notes/v11.4.0.md

```diff
@@ -0,0 +1,77 @@
+sidebar_label: v11.4.0
+image: "img/release-note-link-preview.png"
+---
+
+_July 06th, 2023_
+
+## Console
+
+### New Features
+
+### Improvements
+
+### Bug Fix
+
+## Fast Data
+
+### New Features
+
+#### Single Views and CRUD Collections are now available as sources inside the No Code ER Schema
+
+Single Views and CRUD Collections can now be used as an input for the No Code ER Schema. The new sources will be available in the right panel of the canvas and can be easily added to the canvas with a simple drag-and-drop, just as it is already possible with Projections. When added to the ER Schema, Single Views and CRUD Collections are then included as available dependencies inside the No Code Aggregation.
+
+![Collections inside ER Schema](./img/overview/collections-er-schema.png)
+
+#### Index autogeneration for projections creates indexes for `__internal_counter` and `__internal_counterType`
+
+The Console can now automatically generate the `__internal_counter` and `__internal_counterType` indexes, which have been introduced from `v7.5.0` of the Real-Time Updater in the update Projection queries.
+
+#### The Aggregation config map inside the Single View Creator microservice is automatically handled
+
+When the attaching a Single View Creator to a Single View, the Aggregation config map is now automatically handled by Fast Data. The config map will be editable from the No Code Aggregation screen, without needing to access the Microservice section and the related Single View Creator. Thus, the config map will be displayed as read-only inside this section.
+
+#### Projection changes on Mongo with Single View Trigger Generator
+
+Starting from the `v3.0.2`, Single View Trigger Generator now supports projection changes from Mongo. Previously, the SVTG could only trigger the generation of new Single Views by sending messages onto Kafka following the `sv-trigger` format. Now, Projections Changes are generated onto the MongoDB database.
+
+#### Other improvements and changes
+
+* the UX has been improved inside the Dependency editor modal in the No Code Aggregation page with improved warnings and placeholders
+* the Single View Creator column inside the Single Views page now includes links to each Single View Creator detail page instead of redirecting to the Microservices section
+
+### Bug fix
+
+This version addresses a bug inside the Dependency editor modal in the No Code Aggregation, which allowed the selection of Source Projections with one-to-Many condition from every configuration and not only from their parent.
+
+Some bugs are fixed in the @mia-platform-internal/single-view-creator-lib. When Kafka fails for a faulty message, the healthiness and readiness routes now return KO; also, error logging has been fixed.
+
+Real-Time Updater now properly supports Debezium CDC change events with and without the change schema. In fact, _kafka-importer-lib_ has been upgraded to `v7.5.3` to fix the Debezium Message Adapter implementation.
```

Comment:
The user does not know about the existence of the kafka importer lib, no need to mention it

Reply:
You're totally right, updated!